inception-project/inception
# #4175 - Disable binary CAS format support by default

**What's in the PR**
- Disable format support by default and update documentation

**How to test manually**
* Start INCEpTION and check if the format is available e.g. when importing a document through project settings

**Automatic testing**

**Documentation**
